submitted/wechat-backend/README.md

```diff
@@ -0,0 +1,42 @@
+# WeChat Account Backend
+
+[wechat-backend]https://github.com/jenkins-zh/wechat-backend) is a backend api server for the WeChat Account.
+
+## Introduction
+
+This chart helps you run a backend api server. This server could allow you to manage your wechat account easily.
```

Comment:
How to manage wechat account？ Users would like to see more details or at least a url reference

Reply:
My bad.